library/src/main/java/com/bumptech/glide/load/engine/bitmap_recycle/BitmapCopier.java

```diff
@@ -0,0 +1,34 @@
+
+package com.bumptech.glide.load.engine.bitmap_recycle;
+import android.graphics.Bitmap;
+import android.graphics.Matrix;
+import androidx.annotation.Nullable;
+/**
```

Comment:
please can you run this file through google-java-format, the whitespace in the header is not consistent

Reply:
Done.